pkg/mover/chart.go

```diff
@@ -230,15 +224,65 @@ func (cm *ChartMover) Print() {
 	cm.printMove()
 }
 
-func (cm *ChartMover) loadChart(chartPath string) error {
+func (cm *ChartMover) loadChart(src *Source) error {
 	var err error
```

Comment:
this variable declaration might not be needed if you `if cm.chart, err = loader.Load(src.Chart.Local.Path); err != nil`

Reply:
will review this whole construct.

Reply:
Fixed now all code paths directly return something:
~~~go
func (cm *ChartMover) loadChart(src *Source) error {
	if src.Chart.Local != nil {
		return cm.loadChartFromPath(src.Chart.Local.Path)
	} else if src.Chart.IntermediateBundle != nil {
		return cm.loadChartFromIntermediateBundle(src.Chart.IntermediateBundle.Path)
	}
	return fmt.Errorf("must provide either a local chart or an intermediate bundle as input")
}
~~~